lib/IDE/CodeCompletion.cpp

```diff
@@ -2180,9 +2191,6 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
       return SemanticContextKind::Local;
 
     case DeclVisibilityKind::MemberOfCurrentNominal:
-      if (IsSuperRefExpr &&
-          CurrentMethod && CurrentMethod->getOverriddenDecl() == D)
-        return SemanticContextKind::ExpressionSpecific;
```

Comment:
Doesn't really impact this patch, but do you know why this check was on `MemberOfCurrentNominal` and not `MemberOfSuper`?

Reply:
This is for prioritizing the overridden method after `super.`
```
class Base {
  func foo() {} 
}
class Sub: Base {
  func foo() { super.#^HERE^# }
}
```
In this case, the base type for `lookupVisibleMemberDecls` is `Base` not `Sub`, so `Base.foo()` is found as `MemberOfCurrentNominal` of `Base`.

But I just noticed this mechanism doesn't work for:
```
class Base {
  func foo() {} 
}
class Sub: Base { }
class Sub2: Sub {
  func foo() { super.#^HERE^# } 
}
```
😅

Reply:
So my change unintentionally fixed this issue.
I added a few cases to `test/IDE/complete_after_super.swift`